docs/understanding-tracking-design/versioning-your-data-structures/index.md

```diff
@@ -14,29 +14,8 @@ As well as good practice, versioning has an important role in telling Snowplow L
 
 For example, for certain changes there will be a need to create new columns, update columns or even create whole new tables. For this reason, it's important you understand when your change is breaking and version correctly.
```

Comment:
Once again, I think this should be kept as a single page, perhaps with tabs for Cloud and Enterprise and some common text.

Reply:
We can agree that these sections need reviewing, but this content as it is does not entirely apply to Cloud customers. And I am wary of just rashly splitting it up. The docs need a better strategy to handle Cloud and Enterprise experiences.